bigbone/src/main/kotlin/social/bigbone/api/entity/Report.kt

```diff
@@ -17,5 +17,63 @@ data class Report(
      * Whether an action was taken yet.
      */
     @SerializedName("action_taken")
-    val actionTaken: String = ""
-)
+    val actionTaken: Boolean = false,
```

Comment:
I mentioned that in #271 and wanted to do that in #20. Good that you’ve changed it already! 👍 

@andregasser I have never worked on any library. Is this considered a breaking change and should this be mentioned somewhere somehow?

Reply:
IMHO, it is ok to change the data type now. I did not release an official version yet, just pushing out snapshot builds. Now we can still to this kind of stuff without issues. I will label this PR and the issue with the breaking change label, so hopefully, once I build the next snapshot, it reminds me to put a note into the release notes. :-)